src/main/java/io/github/mapepire_ibmi/types/DaemonServer.java

```diff
@@ -49,24 +49,56 @@ public DaemonServer() {
 
     }
 
+    /**
+     * Construct a new DaemonServer instance.
+     * 
+     * @param host     The hostname or IP address of the server.
+     * @param port     The port number to connect to.
+     * @param user     The username for authentication.
+     * @param password The password for authentication.
+     */
+    public DaemonServer(String host, int port, String user, String password) {
```

Comment:
Recommend to use char[] instead of string for password, and set it to blank once it's not used anymore to make it more secure.

Reply:
For consistency with the other clients, I think keeping it as a string is best. As for setting it to blank, all clients leave it up to the application to manage this `DaemonServer` object. We won't really know when this object is no longer used so it is up to the application to do this.